postgres/postgres_test.go

```diff
@@ -54,6 +54,22 @@ func TestGetInstance(t *testing.T) {
 	})
 }
 
+func TestGetInstances(t *testing.T) {
+	t.Parallel()
+	Convey("When a request for all, a list of instances are returned", t, func() {
+		jsonContent := "{ \"state\":\"Created\"}"
```

Comment:
I changes the tests to lower case, I think we need to force lower case characters when a request comes in. I Added a TODO